pandas/core/frame.py

```diff
@@ -3822,10 +3822,13 @@ def _apply_standard(self, func, axis, ignore_failures=False, reduce=True):
         if reduce:
 
             try:
-
                 # the is the fast-path
                 values = self.values
-                dummy = Series(NA, index=self._get_axis(axis),
+                # Create a dummy Series from an empty array
+                # Unlike filling with NA, this works for any dtype
```

Comment:
you need to add a test for this change here, pls give me an example of the issue